keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -715,30 +717,25 @@ to describe a selector based policy that applies to a multi-cluster service.
 
 _Optional, but recommended._
 
-MCS aims to align with the existing [service DNS
+The full specification for Multicluster Service DNS is in this KEP's [specification.md](specification.md). MCS aims to align with the existing [service DNS
 spec](https://github.com/kubernetes/dns/blob/master/docs/specification.md). This
-section assumes familiarity with in-cluster Service DNS behavior.
-```
-<<[UNRESOLVED]>>
-Full DNS spec needed prior to Beta graduation following:
-https://github.com/kubernetes/dns/blob/master/docs/specification.md
-<<[UNRESOLVED]>>
-```
-When a `ServiceExport` is created, this will cause a domain name for the
-multi-cluster service to become accessible from within the clusterset. The
-domain name will be `<service>.<ns>.svc.clusterset.local`. 
+section provides an overview of the multicluster DNS specification and its rationale, and assumes familiarity with in-cluster Service DNS behavior.
 
-**ClusterSetIP services:** Requests to this domain name from within an importing
-cluster will resolve to the clusterset IP, which points to endpoints for pods
-within the underlying `Service`(s) across the clusterset.
+In short, when a `ServiceExport` is created, this will cause a domain name for the
+multi-cluster service to become accessible from within the clusterset. The
+domain name will be `<service>.<ns>.svc.clusterset.local`. This domain name operates differently depending on whether the `ServiceExport` refers to a ClusterSetIP or Headless service:
 
-**Headless services:** Within an importing cluster, the clusterset domain name
+  * **ClusterSetIP services:** Requests to this domain name from within an importing
+cluster will resolve to the clusterset IP. Requests to this IP will be load-balanced
+across all endpoints exported with `ServiceExport`s across the clusterset.
+  * **Headless services:** Within an importing cluster, the clusterset domain name
 will have multiple `A`/`AAAA` records, each containing the address of a ready
 endpoint of the headless service. `<service>.<ns>.svc.clusterset.local` will
-resolve to the set of all ready pod IPs for the service.
+resolve to the entire set or the subset of ready pod IPs, depending on the implementation
```

Comment:
This specifically says "ready".   Single-cluster is moving to a slightly different definition - "ready" IPs when possible, but if there are no "ready" IPs then terminating-but-serving IPs will be used instead.  We don't need to adopt that here, but we should consider it as a future step, even if just for conceptual alignment.

Reply:
Got it -- resolving this here for now and let's plan to do it in the single-cluster DNS spec and this DNS spec at the same time (or it might come "for free" since we utilize single-cluster specification's definition of "ready", but may want to bump the schema or otherwise comms that in this doc explicitly) cc @johnbelamaric